### PR TITLE
DAMIEN-724 - Fix Cancel Button Dark Mode issues

### PR DESCRIPTION
**Pull request by mohamalnadi**
Jira: https://jira-secure.berkeley.edu/browse/DAMIEN-724

### CODE REVIEW

File: src/components/evaluation/EvaluationTable.vue [hunk 1]
```diff
@@ -1054,4 +1054,7 @@ tr.border-top-none td {
 .xlisting-note {
   font-size: 0.8em;
 }
+.evaluation-form-btn-cancel {
+  color: rgba(var(--v-theme-on-surface),var(--v-high-emphasis-opacity)) !important;
+}
```
Comment (johncrossman) [inline src/components/evaluation/EvaluationTable.vue:1059]: @mohamalnadi - This makes sense to me. Can we make it a reusable style in `global.css`? That'll give us an easy-to-use best practice for all such buttons.

Comment (lyttam) [reply src/components/evaluation/EvaluationTable.vue:1059]: Sorry, I merged too soon. I agree, but do we actually need a style for it or would just removing `text-primary` be enough?

Comment (mohamalnadi) [reply src/components/evaluation/EvaluationTable.vue:1059]: This was a really weird issue that was difficult for me to pinpoint exactly. Removing text-primary made it work in dark mode, but then the text was white in light mode (and the background of the button was also white). I went through the chrome inspector CSS and noticed that the button had a `color: inherit` style which caused the issue. The `color: inherit` seems to be coming from Vuetify, but I am not sure. This CSS style overrides it and fixes the issue.

I am worried putting this in the global.css might cause more issues? I am not sure, but I think this is the only button we have found with such an issue.

**Comment by lyttam** [comment]
That makes sense @mohamalnadi. Perhaps we can do a review of all the button styles in Damien at some point to look for inconsistency.

**Pull request merged**